x-pack/plugins/stack_connectors/public/connector_types/crowdstrike/crowdstrike.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import type {
+  ActionTypeModel as ConnectorTypeModel,
+  GenericValidationResult,
+} from '@kbn/triggers-actions-ui-plugin/public';
+import {
+  CROWDSTRIKE_CONNECTOR_ID,
+  CROWDSTRIKE_TITLE,
+  SUB_ACTION,
+} from '../../../common/crowdstrike/constants';
+import type {
+  CrowdstrikeConfig,
+  CrowdstrikeSecrets,
+  CrowdstrikeActionParams,
+} from '../../../common/crowdstrike/types';
+
+interface ValidationErrors {
+  subAction: string[];
+}
+
+export function getConnectorType(): ConnectorTypeModel<
+  CrowdstrikeConfig,
+  CrowdstrikeSecrets,
+  CrowdstrikeActionParams
+> {
+  return {
+    id: CROWDSTRIKE_CONNECTOR_ID,
+    actionTypeTitle: CROWDSTRIKE_TITLE,
+    iconClass: lazy(() => import('./logo')),
+    isExperimental: true,
+    selectMessage: i18n.translate(
+      'xpack.stackConnectors.security.crowdstrike.config.selectMessageText',
+      {
+        defaultMessage: 'Execute CrowdStrike scripts',
```

Comment:
"scripts" sounds strange to me. Maybe set the message to "Execute CrowdStrike response actions" ??